lib/API/JSON/v1/Newsletters.php

```diff
@@ -537,7 +537,7 @@ function listing($data = []) {
         $queue
       );
 
-      $data[] = $this->wp->applyFilters('mailpoet_api_newsletters_listing_item', $newsletter->asArray());
```

Comment:
I don't think we should remove this filter. Someone can be still using it, don't you think?

Reply:
@pavel-mailpoet Hmm, you mean third party plugins? Because according to Git history this filter was added only to perform that side-effect. Honestly, I'd like to get rid of all filters that can modify our API data or anything. It is a very ugly and dangerous way of extending stuff. I understand when a filter changes some parameters but whole API data, I don't know.

Back to this filter - there's no way I could ensure nobody's using it. Do you think we have to keep it?

Reply:
It could be a 3rd party plugin, or it could be a customer who can have a piece of code that is using it. My understanding is we cannot remove any filters because we don't know who is using them.